packages/requisitions/src/RequestRequisition/DetailView/columns.ts

```diff
@@ -83,14 +74,36 @@ export const useRequestColumns = () => {
             itemStats;
 
           const monthsString = availableMonthsOfStockOnHand
-            ? `(${availableMonthsOfStockOnHand.toFixed(2)} ${t('label.months', {
-                count: availableMonthsOfStockOnHand,
-              })})`
+            ? `(${formatNumber.round(availableMonthsOfStockOnHand, 1)} ${t(
```

Comment:
`formatNumber` handles intl so will give `3.67` if language is english and `3,67` if you speak French